src/main/cli/settings.c

```diff
@@ -1714,6 +1714,7 @@ const clivalue_t valueTable[] = {
 
 // PG_POSITION
     { "position_alt_source",           VAR_INT8   | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_POSITION_ALT_SOURCE }, PG_POSITION, offsetof(positionConfig_t, altSource) },
+    { "position_gps_alt_min_sats",     VAR_INT8   | MASTER_VALUE, .config.minmaxUnsigned = { 0, 50 }, PG_POSITION, offsetof(positionConfig_t, gpsAltMinSats) },
```

Comment:
I think is better to start with `position_alt` like the other parameter. The minimum number I think must be bigger, we will not have any lock with altitude without 4 satelites at least:
```suggestion
    { "position_alt_gps_min_sats",     VAR_INT8   | MASTER_VALUE, .config.minmaxUnsigned = { 4, 50 }, PG_POSITION, offsetof(positionConfig_t, gpsAltMinSats) },
```
We have another parameter that specifies the minimum satellites `gps_rescue_min_sats`, the objective of both is the same: define the minimum number of satellites to decide a good lock. I understand the creation of this new because the name of the old can be confusing, but I'm not sure is the best option. Some other opinions?

Reply:
We had discussed it in #11342 with @ledvinap - decided to remove config option and hardcode some value (by my experience - a good value is 10 sats (6 is not enough - altitude jumps are still high).
Also I made a little refactoring there, I will move code here and update PR.

Reply:
Another proposal would be to further rearrange the parameter name to
```suggestion
    { "position_alt_gps_sats_min",     VAR_INT8   | MASTER_VALUE, .config.minmaxUnsigned = { 0, 50 }, PG_POSITION, offsetof(positionConfig_t, gpsAltMinSats) },
```
but I'm pretty pedantic, so choose what you like better

Reply:
I don't like to hardcode the value. We can use a "good" default value, but hardcode it does not seem a good choice to me.

Reply:
I'll bring it back. IMO both options are ok. IMO the whole estimation code should be refactored, currently just a quick fix to get rid of such strong altitude mismatch.

Reply:
Fixed (actually in refactored version I added two variables - one for obtaining GPS sats and another one for losing. 
We can't use the same number - in this case we will get altitude fluctuations in the period when receiving and losing 1-2 gps sats (this is the real world behavior of gps - it can add a few sats, and then lose 1 or 2, and then get them back).
Fallback was requested in #11342.